zerver/views/scheduled_messages.py

```diff
@@ -45,6 +53,17 @@ def delete_scheduled_messages(
     return json_success(request)
 
 
+@typed_endpoint
+def delete_reminders(
```

Comment:
This should be singular I think? `delete_scheduled_messages` has the same mistake, you can fix that in a prep commit.